README.md

```diff
@@ -42,6 +42,8 @@ $ docker build --rm=true --tag=sonatype/nexus .
 
 ## Notes
 
+* The UI is accessible at: http://localhost:8081/nexus/
```

Comment:
Do they have the opportunity to change/configure the port?

Reply:
Yeah, they can always override. But this is the default. If they overrode the port then they would know to use it.